stable/wordpress/templates/svc.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       {{- end }}
     - name: https
       port: {{ .Values.service.httpsPort }}
-      targetPort: https
+      targetPort: {{ default "https" .Values.service.httpsTargetPort }}
```

Comment:
Why not `targetPort: {{ .Values.service.httpsTargetPort }}`

Reply:
Left it in there but it's redundant now with the changes to the values file. Will remove the default.